crawler/test_undetected.py

```
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+"""
+Test script to verify undetected Playwright is working correctly.
+This script tests common detection vectors that anti-bot systems use.
+"""
+
+import asyncio
+from playwright.async_api import async_playwright
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)